docs/_config.yml

```diff
@@ -0,0 +1,56 @@
+# Welcome to Jekyll!
+#
+# This config file is meant for settings that affect your whole blog, values
+# which you are expected to set up once and rarely edit after that. If you find
+# yourself editing this file very often, consider using Jekyll's data files
+# feature for the data you need to update frequently.
+#
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'bundle exec jekyll serve'. If you change this file, please restart the server process.
+#
+# If you need help with YAML syntax, here are some quick references for you: 
+# https://learn-the-web.algonquindesign.ca/topics/markdown-yaml-cheat-sheet/#yaml
+# https://learnxinyminutes.com/docs/yaml/
+#
+# Site settings
+# These are used to personalize your new site. If you look in the HTML files,
+# you will see them accessed via {{ site.title }}, {{ site.email }}, and so on.
+# You can create any custom variable you would like, and they will be accessible
+# in the templates via {{ site.myvariable }}.
+
+title: Moov ImageCashLetter
+email: security@moov.io
+description: >- # this means to ignore newlines until "baseurl:"
+  Moov ImageCashLetter implements a low-level Image Cash Letter (ICL) interface for parsing, creating, and validating ICL files. Moov ImageCashLetter exposes an HTTP API for REST-based interaction.
+url: "https://moov-io.github.io" # the base hostname & protocol for your site, e.g. http://example.com
+baseurl: "/imagecashletter" # the subpath of your site, e.g. /blog
+twitter_username: jekyllrb
```

Comment:
Should these values be Moov specific?

Reply:
`twitter_username` and `github_username` are actually for the default template, so they don't do anything for us. I'll just remove them.

Reply:
Oh ok. Sounds good.